### PR TITLE
zram-swap: dependency fix, stats info output

### PR DESCRIPTION
**Pull request by vortigont**
**zram-swap:**
- fix dependency on BUSYBOX_CONFIG_SWAPONOFF (removed in 84da2a6)
- add busybox defaults checking (fix zram-swap always installs swap-utils and libblkid as dependency, even if busybox includes mkswap by default)
- Add zram compaction and statistics info output
    Executing '/etc/init.d/zram start' during runtime (with a swap being already mounted) triggers zram device compaction and prints out nice stats info about zram memory usage

**busybox:**
- enable swapon/off by default to make it consistent with mkswap
      No size increase on busybox binary.
      Since busybox mkswap is already enabled by default it seems reasonable to enable swapon/off too. For ex. this obsoletes installing block-mount dependency for zram-swap.

build/run tested on ramips/ar71

Stats info example

```
Compacting zram device...
Memory usage reduced by   - 0.8 Mb
Overhead reduced by       - 37.7 %
Gathering stats info for zram device "zram0"

Z-RAM
-----
Block device              - /sys/block/zram0
Device size               - 64 Mb
Compression algo          - lzo [lz4] deflate 
Compression streams       - 4

DATA
----
Original data size        - 40.55 Mb
Compressed data size      - 16.60 Mb
Compress ratio            - 2.44 

MEMORY
------
Memory used, total        - 17.89 Mb
Allocator overhead        - 1.29 Mb
Allocator efficiency      - 92.80 %
Maximum memory ever used  - 28.73 Mb
Memory limit              - 0.00 Mb

PAGES
-----
Same pages count          - 198
Pages compacted           - 200
Free pages discarded      - 22779
```

### CODE REVIEW

File: package/system/zram-swap/files/zram.init [hunk 1]
```diff
@@ -106,14 +106,70 @@ zram_comp_streams()
...
 	if [ $( grep -cs zram /proc/swaps ) -ne 0 ]; then
 		logger -s -t zram_start -p daemon.notice "[OK] zram swap is already mounted"
-		return 1
+		# If not running interactively, than just quit
+		[ -z "$PS1" ] && return 1
+
+		# show memory stats and compact all zram swaps
+		for zram_dev in $( grep zram /proc/swaps |awk '{print $1}' ); do {
+			zram_compact "$zram_dev"
+			zram_stats "$zram_dev"
```
Comment (hauke) [inline package/system/zram-swap/files/zram.init]: I would prefer if you add an extra command like it is done for the dsl:
```
EXTRA_COMMANDS="status lucistat"
EXTRA_HELP="	status  Get DSL status information
	lucistat  Get status information if lua friendly format"
```
here:
package/network/config/ltq-vdsl-app/files/dsl_control
target/linux/lantiq/base-files/lib/functions/lantiq_dsl.sh

like this:
```
EXTRA_COMMANDS="compact status"

status() {
for zram_dev in $( grep zram /proc/swaps |awk '{print $1}' ); do {
#print various stats info about zram swap device
	local zdev="/sys/block/$( basename "$1" )"
.....
}
```

File: package/utils/busybox/Config-defaults.in [hunk 2]
```diff
@@ -1728,16 +1728,16 @@ config BUSYBOX_DEFAULT_SETSID
 	default n
 config BUSYBOX_DEFAULT_SWAPON
 	bool
-	default n
+	default y
```
Comment (hauke) [inline package/utils/busybox/Config-defaults.in]: Please check how much the size of the busybox binary increases when you activate these options on a common target.

File: package/system/zram-swap/files/zram.init (hunk 1 above)
Comment (vortigont) [reply package/system/zram-swap/files/zram.init]: Sounds good, I'll rework it.

File: package/utils/busybox/Config-defaults.in (hunk 2 above)
Comment (vortigont) [reply package/utils/busybox/Config-defaults.in]: No increase in binary size. Checked for ramips/AR.

File: package/system/zram-swap/files/zram.init (hunk 1 above)
Comment (vortigont) [reply package/system/zram-swap/files/zram.init]: Done

**Comment by vortigont** [comment]
@hauke a kind reminder to review :) tnx!

**Comment by CodeFetch** [comment]
@vortigont I haven't checked it... Mb means Megabit. Is this actually Mb or MiB?

**Comment by vortigont** [comment]
@CodeFetch it's MiB - 2^20 bytes, although I never liked this abbreviation :)

**Comment by chunkeey** [comment]
Merged. Thanks.

**Pull request closed without merging**